### PR TITLE
Add test - Check product block in order page BO

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.7.x
| Description?      | Check product block in order page BO
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| How to test?      | TEST_PATH="functional/BO/02_orders/01_orders/viewAndEditOrder/02_productBlock.js" URL_FO=yourShopURL npm run specific-test
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/23452)

### CODE REVIEW

File: tests/UI/pages/BO/orders/view.js [hunk 1]
```diff
@@ -358,6 +417,147 @@ class Order extends BOBasePage {
...
+    return this.getNumberFromText(page, this.productsCountSpan);
+  }
+
+  /**
+   * Search product
+   * @param page
+   * @param name
+   * @returns {Promise<void>}
+   */
+  async SearchProduct(page, name) {
```
Comment (Progi1984) [inline tests/UI/pages/BO/orders/view.js]:
```suggestion
  async searchProduct(page, name) {
```

**Review by Progi1984 — COMMENTED** [review]
A small feedback

File: tests/UI/pages/BO/orders/view.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/pages/BO/orders/view.js]: Done thank you :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**